include/Request/CustomTaxonomyAction.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * Custom Taxonomy metabox's Ajax Action Class.
+ *
+ * @since 2.0.0
+ */
+class BM_Request_CustomTaxonomyAction implements BM_Loadie {
+
+	/**
+	 * The BM_Loadie() calls this method.
+	 *
+	 * @since 2.0.0
+	 */
+	public function load() {
+		add_action( 'wp_ajax_load_custom_taxonomy_by_post_type', array( $this, 'load_custom_taxonomy_by_post_type' ) );
+		add_action( 'wp_ajax_load_custom_terms_by_taxonomy', array( $this, 'load_custom_terms_by_taxonomy' ) );
+	}
+
+	/**
+	 * Loads the custom Taxonomy by Post Type.
+	 *
+	 * @since 2.0.0
+	 */
+	public static function load_custom_taxonomy_by_post_type() {
+		/* The action should be in the {page_slug}-nonce format.
+		 * {page_slug} is the slug of the page on which the AJAX is requested.
+		 */
+		check_ajax_referer( 'bulk-move-posts-nonce', 'nonce' );
```

Comment:
Don't hardcode the nonce name. Build it dynamically from page slug. Also move this logic into a common function.

Reply:
> Don't hardcode the nonce name.

https://github.com/sudar/bulk-move/blob/43-feature/refactor-move-by-taxonomy/include/BulkMove.php#L179

From the above line, it looks like we have to retrieve the page slug by hard-coding the slug. This made me use the slug directly. Once we have a key to indicate different pages, we could refactor it.

> Build it dynamically from page slug. Also move this logic into a common function.

I removed the `check_admin_referrer()` in AJAX requests and used the one in `Page/Base.php`

https://github.com/sudar/bulk-move/blob/43-feature/refactor-move-by-taxonomy/include/Page/Base.php#L130-L154